starlite/plugins/sql_alchemy/config.py

```diff
@@ -18,8 +18,9 @@
 from starlite.config.logging import BaseLoggingConfig, LoggingConfig
 from starlite.exceptions import MissingDependencyException
 from starlite.types import BeforeMessageSendHookHandler
-from starlite.utils import AsyncCallable, default_serializer
+from starlite.utils import default_serializer
 
+from ...datastructures.async_callable import AsyncCallable
```

Comment:
3rd level relative import is quite ugly. Probably should be `from starlite.datastructures import AsynCallable`

Reply:
fixed. This is the IDE.